cypress/integration/portal-dashboard-smoke-test.spec.js

```diff
@@ -12,4 +12,10 @@ context("Portal Dashboard UI",()=>{
             });
         });
     });
+    describe('Number of students in class',()=>{
+        let numStudents = 6; //TODO: it would be better if numStudents is not hard coded.
+        it('verify number of students in a class loads',()=>{
+            cy.get('[data-test=num-student-container]').should('be.visible').and('contain', numStudents+ ' students');
```

Comment:
Sam mentioned this in my PR so I'll ask the same question.  Most of the project uses `data-cy` instead of `data-test`.  Do you think we should use `data-cy` to be consistent?

Reply:
Yes! I got so used to putting data-test, forget this one uses this one used data-cy

Reply:
I'm going to change them to `data-cy` in my PR as well then.